utility_code/minecraft/region.py

```diff
@@ -274,7 +290,7 @@ class Region(BaseRegion):
     """A 'region' type region file"""
 
     @classmethod
-    def folder_name(self):
+    def folder_name(cls):
```

Comment:
While this is an improvement, I'm wondering if we can get away with making this a `@staticmethod` instead (no self/class passed in).